docs/docs/faq.md

```diff
@@ -80,9 +80,13 @@ Please note, many keyboards only have a single PCB which includes the “brains
 
 Currently, ZMK only supports wireless split, but wired split is possible and we welcome contributions!
 
+### How is the latency?
+
+ZMK uses [interrupts](https://en.wikipedia.org/wiki/Interrupt) to scan for key presses. Thus, the polling speed is high (> 1ms) and lowers input delay. For more information on ZMK's latency, check out this [video](https://www.youtube.com/watch?v=jWL4nU-vtWs).
```

Comment:
```suggestion
ZMK uses [interrupts](https://en.wikipedia.org/wiki/Interrupt) to scan for key presses. Thus, the polling speed is high (> 1ms) and lowers input delay. For a comparison of input latency between ZMK and other keyboard offerings, see this [video](https://www.youtube.com/watch?v=jWL4nU-vtWs).
```

Alternatively, similar to https://zmk.dev/docs/features/underglow:

```suggestion
ZMK uses [interrupts](https://en.wikipedia.org/wiki/Interrupt) to scan for key presses. Thus, the polling speed is high (> 1ms) and lowers input delay. For a comparison of input latency between ZMK and other keyboard offerings, see the following video:

<figure class="video-container">
  <iframe src="//www.youtube.com/embed/2KJkq8ssDU0" frameborder="0" allowfullscreen width="100%"></iframe>
</figure>
```

Some keyboards (BDN9 Rev. 2, Ferris Rev. 1/2, Planck, Clueboard) use polling instead of interrupts, so this section may not be completely accurate.